src/backend/defs/rest-api/objects/NewDevice.json

```diff
@@ -0,0 +1,18 @@
+{
+  "type": "object",
+  "properties": {
+    "name": {
+      "type": "string"
+    },
+    "type": {
```

Comment:
may be an enum ? 
does type mean [desktop|smartphone|tablet|watch|washing machine] ?

later it would be nice to have OS and version.